environment.py

```diff
@@ -362,11 +364,20 @@ def env() -> Mapping[str, Optional[str]]:
         # Typically only set for main deployments.
         'AZUL_ENABLE_MONITORING': '0',
 
-        # The default bundle UUID prefix to use for reindexing bundles in the DSS
-        # and for subscriptions to the DSS. If this variable is set to a non-empty
-        # string, only bundles whose UUID starts with the specified string will be
-        # indexed.
-        'AZUL_DSS_QUERY_PREFIX': '',
+        # Identifies the DSS repository endpoint and prefix to index.
+        # The syntax in EBNF is:
+        #
+        # source = 'https://', endpoint
+        #          ':', [ UUID prefix [ '/', partition prefix length ] ] ;
+        #
+        # `endpoint` is the URL of the DSS instance. For `UUID prefix` and
```

Comment:
The way you modeled this, `source` already is a URL. If it is, endpoint can't be a URL, too, because that would mean `https://https://foo.com/` would be valid.